web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -419,12 +437,166 @@ export function ServerAccessSpecSection({
   );
 }
 
-function KubernetesAccessSpecSection() {
-  // TODO(bl-nero): add the Kubernetes section
-  return null;
+export function KubernetesAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<KubernetesAccessSpec>) {
+  return (
+    <>
+      <FieldSelectCreatable
+        isMulti
+        label="Groups"
+        isDisabled={isProcessing}
+        formatCreateLabel={label => `Group: ${label}`}
+        components={{
+          DropdownIndicator: null,
+        }}
+        value={value.groups}
+        onChange={groups => onChange?.({ ...value, groups })}
+      />
+
+      <Text typography="body3" mb={1}>
+        Labels
+      </Text>
+      <LabelsInput
+        disableBtns={isProcessing}
+        labels={value.labels}
+        setLabels={labels => onChange?.({ ...value, labels })}
+      />
+
+      <Flex flexDirection="column" gap={3} mt={3}>
+        {value.resources.map((resource, iRes) => (
```

Comment:
Nit, but why `iRes` and not `index` or `resourceIndex`? When I saw `resources.map((resource, iRes) => …`, I assumed that `resources` is a map because I didn't quite understand at first what `iRes` is.